Project.toml

```diff
@@ -0,0 +1,7 @@
+name = "MathOptInterface"
+uuid = "b8f27783-ece8-5eb3-8dc8-9495eed66fee"
+version = "0.6.2"
+
```

Comment:
Missing a [compatibility](https://docs.julialang.org/en/v1/stdlib/Pkg/#Compatibility-1) section specifying the allowed versions of Compat and julia.